client/src/modules/employees/registration/employees.html

```diff
@@ -197,6 +179,36 @@
                     </div>
                   </div>
 
+
+                  </fieldset>
+
+                  <fieldset>
+                    <legend translate>PATIENT_REG.PATIENT_DETAILS</legend>
+
+                     <!-- Employee Hospital File Number -->
+                  <div
+                    class="form-group has-feedback"
+                    ng-class="{'has-error' : DetailForm.hospitalNumber.$invalid && EmployeeRegistrationForm.$submitted || DetailForm.hospitalNumber.$error.unique}">
```

Comment:
It would be much clearer to include brackets here so that we know how the logic works.   I believe it works like this:
```html
ng-class="{'has-error' : (DetailForm.hospitalNumber.$invalid && EmployeeRegistrationForm.$submitted) || DetailForm.hospitalNumber.$error.unique}"
```
but you should check to make sure.